types/index.d.ts

```diff
@@ -1,21 +1,84 @@
+/**
+ * A database transaction
+ */
 export type Transaction = Array<object>;
+/**
+ * The Schema is a way to to simplify relational queries at query time.
+ * Presently, it allows you to define relationships and uniqueness constraints.
+ */
 export type Schema = object;
+/**
+ * Homebase configuration.
+ */
 export type config = {schema?: Schema, initialData?: Transaction };
+/**
+ * Entities are JSON objects with extra features.
```

Comment:
I wouldn't call them JSON objects since you can't get attributes off of them without the get function.

Maybe: "A reference to an entities data"

Reply:
Got it we should remove the "Entities are like JSON with bonus features" from the README and docs then

Reply:
IDK, they are "like" JSON objects with extra features, but they are not JSON objects